setup/www/tools/metrics/download-counts.awk

```diff
@@ -63,7 +65,9 @@ BEGIN {
       os = "osx"
     } else if (match(fileType, /^sunos-/)) {
       os = "sunos"
-    } else if (match(fileType, /msi$/) || match(file, /node\.exe$/)) {
+    } else if (match(fileType, /^aix-/)) {
+      os = "aix"
+    } else if (match(fileType, /msi$/) || match(file, /node\.exe$/) || match(fileType, /^win-/)) {
```

Comment:
Thinking yes seeing how path is prepended by `win-{x86|x64}` -- the variable (`fileType`) is just poorly named.